src/Features/LanguageServer/Protocol/Features/Options/AutoFormattingOptionsStorage.cs

```diff
@@ -18,14 +18,14 @@ public static AutoFormattingOptions GetAutoFormattingOptions(this IGlobalOptionS
         };
 
     internal static readonly PerLanguageOption2<bool> FormatOnReturn = new(
-        "FormattingOptions_AutoFormattingOnReturn", AutoFormattingOptions.Default.FormatOnReturn);
+        "csharp_format_on_return", AutoFormattingOptions.Default.FormatOnReturn);
```

Comment:
>csharp_format_on_return [](http://example.com/codeflow?start=9&length=23)

Seems like these should be in C#, but it might be more work to make that change. Perhaps file an issue to follow up?

Reply:
Sure

Reply:
Actually this is in the LSP project. (I know you delete the CSharp.LSP project some time ago) do you want to bring it back?